sympy/core/tests/test_assumptions.py

```diff
@@ -481,6 +477,9 @@ def test_composite():
     assert S(2).is_composite is False
     assert S(17).is_composite is False
     assert S(4).is_composite is True
+    x = Dummy(integer=True, positive=True, prime=False)
+    assert x.is_composite
```

Comment:
@smichr, as I said in the referenced pr, this is wrong!  Counterexample is x=S.One.

While it's ok to do bad reviews (all people do mistakes), you shouldn't silently ignore already mentioned flaws.  @asmeurer, @certik, I'm surprised that you don't care about this...